include/vga.h

```diff
@@ -119,6 +120,9 @@ enum VGAModes {
 	M_ERROR = 1 << 31,
 };
 
+constexpr auto NumCgaColors = 16;
+constexpr auto NumVgaColors = 256;
```

Comment:
Are we doing British or American spelling of color?  You've got a mix of both in this PR.  I'm not going to comment on each one but do a find on "color" and "colour" and be consistent one way or the other.

Reply:
This is pretty annoying... We use British spelling on our website and in our manual. But in the code we use American spelling, so "color" and letter "Z" everywhere because it's too much of a hassle to change it, and for not much benefit. In code the American spelling is "standard" anyway, kind of; all graphics APIs use "color" AFAIK.

So my approach is to suck up to Americans and use American spelling in the code 😄 But in all text meant for human consumption, so READMEs, commit logs, documentation, even log messages to some extent, I use International English spelling.

Important to note that English is the official language in many countries, and they in all these countries they use British spelling, basically. American spelling is for Americans only, so we're using Internal English spelling, which is effectively British spelling, yes 😄